2차/류동준_정수-제곱근-판별.py

```diff
@@ -0,0 +1,9 @@
+def check(n):
+    value = pow(n,0.5)
+    if int(value) == value:
+        return pow(value+1,2)
+    else:
+        return -1
```

Comment:
사실 if - else 보다  `true if else false` 형태가 더 좋은 것인지는 모르겠으나 보통 후자를 선택하더라구요
```suggestion
    return pow(value+1,2) if int(value) == value else -1
```